libs/infra-tracing/src/lib/datadog-tracer.ts

```diff
@@ -1,20 +1,41 @@
 import tracer from 'dd-trace'
 import { isPerson } from 'kennitala'
+import { Span } from 'opentracing'
 
 if (process.env.NODE_ENV !== 'development') {
+  const rewriteUrl = (url: string): string =>
+    url
+      .split('/')
+      .map((segm) => (isPerson(segm) ? '--MASKED--' : segm))
+      .join('/')
+  const shouldMask = (statusCode?: number): boolean =>
+    !statusCode || statusCode < 400
```

Comment:
We need an additional input here which is the magic env variable that will allow this to be possible at all. By default shouldMask should return `true` for all status codes.

Reply:
oh so this shouldn't be always on? talk in person